updating/preparing-eus-eus-upgrade.adoc

```diff
@@ -24,3 +24,22 @@ Verify that no RHEL7 workers are present in the 4.10 cluster before performing a
 * All the clusters might update using EUS channels for a conventional update without pools paused, but only clusters with non control-plane `MachineConfigPools` objects can do EUS-to-EUS update with pools paused.
 
 include::modules/updating-eus-to-eus-upgrade.adoc[leveloffset=+1]
+
+include::modules/updating-eus-to-eus-upgrade-console.adoc[leveloffset=+2]
+
+[role="_additional-resources"]
+[id="additional-resources_updating-eus-to-eus-upgrade-console"]
+== Additional resources
```

Comment:
Since we want the "Additional resources" list to appear as though it's directly following the body content of the preceding `include`'d module, this should use the `.Additional resources` form instead of `== Additional resources`.

See the "Only use `.` formatting (`.Additional resources`) in a module or to follow a module in an assembly" section in https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#additional-resources-sections

Using the `==` form makes it appear in the TOC, which we don't want in this case. Making it a `.Additional resources` just renders a simple non-TOC heading.

Reply:
gotcha :)